recipes/the-silver-searcher/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "the-silver-searcher" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "ff7243863f22ed73eeab6f7a6d17cfff585a7eaa41d5ab3ae4f5d6db97701d5f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/ggreer/{{ name | replace('-', '_') }}/archive/{{ version }}.tar.gz
```

Comment:
I do not understand what you mean by 'static' checksum. Could you please elaborate?